interface/src/app/signature/signature.tpl.html

```diff
@@ -5,12 +5,25 @@
 
 <h3 class="text-warning" ng-bind="ctrl.statusMessage"></h3>
 
+<h4>
+  <participation-type-selector
+      selected-participation-type="ctrl.selectedParticipationType">
+  </participation-type-selector>
+  <br>
+  <br>
+  {{ctrl.selectedParticipationType.description}}
+</h4>
+
+<hr>
+
 <div ng-if="!ctrl.statusMessage">
+
   <div class="container">
     <div class="row">
       <div class="col-xs-6">
-        <high-weight-genes signature-id="{{ctrl.id}}" genes="ctrl.genes">
-        </high-weight-genes>
+        <participatory-genes signature-id="{{ctrl.id}}" genes="ctrl.genes"
+            selected-participation-type="ctrl.selectedParticipationType">
+        </participatory-genes>
```

Comment:
Ok, here is the template where that wrapping I suggested would make this more readable:

```html
        <participatory-genes
          signature-id="{{ctrl.id}}"
          genes="ctrl.genes"
          selected-participation-type="ctrl.selectedParticipationType">
        </participatory-genes>
```

I like this way because it's easier to scan visually.